pkg/kubelet/apis/pods/server.go

```diff
@@ -0,0 +1,508 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
just remove all years from headers. Now header may omit the year and I do not need to comment on the wrong year =)